CHANGELOG.md

```diff
@@ -4,6 +4,22 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.27.0] - 2022-02-13
+
+This is a fairly small release, primarily to bump substrate dependencies to their latest versions.
+
+The main breaking change is fairly small: [#804](https://github.com/paritytech/subxt/pull/804). Here, the `BlockNumber` associated type has been removed from `Config` entirely, since it wasn't actually needed anywhere in Subxt. Additionally, the constraints on each of those associated types in `Config` were made more precise, primarily to tidy things up (but this should result in types more easily being able to meet the requirements here). If you use custom `Config`, the fix is simply to remove the `BlockNumber` type. If you also use the `Config` trait in your own functions and depend on those constraints, you may be able to define a custom `MyConfig` type which builds off `Config` and adds back any additional bounds that you want.
+
+## Added
+
+- Add find last function ([#821](https://github.com/paritytech/subxt/pull/821))
+- Doc: first item is current version comment ([#817](https://github.com/paritytech/subxt/pull/817))
+- Remove sleeps from release checklist; no longer needed ([#803](https://github.com/paritytech/subxt/pull/803))
```

Comment:
super small nit: should this be placed under `changed`?

Reply:
Yup you aren't wrong :)

Reply:
Yeah, I was on the verge of doing that as I did with the dependabot things; there is just so little to report that I didn't bother! I'll juet remove it entirely.